src/commands/toxics.ts

```diff
@@ -0,0 +1,43 @@
+// Dependencies
+import { Telegraf, ContextMessageUpdate } from 'telegraf'
+import { IUser, getUser, getUserInfo } from '../models'
+import { isReply } from './middleware'
+import { transfer } from './transfer'
+
+function isToxicMinisters(user: IUser) {
```

Comment:
Rename to `isToxicMinister`

Reply:
Also, middleware is written differently. Please, consider looking at the `middleware.ts` for examples